From 77549d99857b96c4e1ae14337586d33819575cd5 Mon Sep 17 00:00:00 2001
From: James Buren <ryuo@frugalware.org>
Date: Sun, 21 Apr 2013 23:38:45 -0500
Subject: [PATCH] Makefile: create the skeleton directory for /etc/ld.so.conf.d
 Why not? We create all the other directories used in our /etc/ld.so.conf
 defaults.

---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index 596cc3d..c61cbeb 100644
--- a/Makefile
+++ b/Makefile
@@ -54,6 +54,7 @@ install:
 	done
 	$(INSTALL) -d $(DESTDIR)/etc/skel
 	$(INSTALL) -d $(DESTDIR)/etc/sysconfig
+	$(INSTALL) -d $(DESTDIR)/etc/ld.so.conf.d
 	$(INSTALL) -d $(DESTDIR)/home
 	$(INSTALL) -d $(DESTDIR)/mnt
 	$(INSTALL) -d $(DESTDIR)/media
-- 
1.8.1.5

